### PR TITLE
Update bglibpy version

### PR DESCRIPTION
**Pull request by wvangeit**

### CODE REVIEW

**Comment by wvangeit** [comment]
Pinging @wizmer

**Review by wizmer — APPROVED** [review]

File: var/spack/repos/builtin/packages/py-bglibpy/package.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class PyBglibpy(PythonPackage):
     git = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
 
     version('develop', branch='master')
+    version('4.1.2', commit='d01c9359edf92e39be5b840b5bb975da1c960988')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-bglibpy/package.py:16]: Tags?

**Comment by wvangeit** [comment]
@pramodk @matz-e 
It looks like the jenkins plans for spack are failing / stuck, is there anything broken ?

File: var/spack/repos/builtin/packages/py-bglibpy/package.py (hunk 1 above)
Comment (wvangeit) [reply var/spack/repos/builtin/packages/py-bglibpy/package.py:16]: Please, let's not start this discussion agaaiin ... We need this fairly urgently...

**Comment by matz-e** [comment]
> @pramodk @matz-e
> It looks like the jenkins plans for spack are failing / stuck, is there anything broken ?

Everything is fine. Jenkins is currently building stuff. I'll see if I can interrupt that safely to get this going.

**Pull request merged**